plugins/modules/postgresql_user.py

```diff
@@ -213,13 +213,19 @@
     name: django
     comment: This is a test user
 
-# Connect to default database, create rails user, set its password (MD5-hashed),
+# Connect to default database, create rails user, set its password (MD5- or sha256-hashed),
 # and grant privilege to create other databases and demote rails from super user status if user exists
+# for SCRAM-SHA-256 you can create the hash by creating a dummy user with the same password & copying
```

Comment:
while this is true, it seems wastful. there's a python script [here](https://gist.github.com/jkatz/e0a1f52f66fa03b732945f6eb94d9c21) by a PG author that can be used instead

Reply:
maybe we could just remove this statement?

Reply:
@vrms thoughts ^

Reply:
removing based on Matrix discussion